pkg/ottl/README.md

```diff
@@ -1,5 +1,13 @@
 # OpenTelemetry Transformation Language
 
+| Status        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Any thoughts around making this an auto-generated section?

```
<!-- status autogenerated section -->
```

Reply:
Had to modify mdatagen, but its done.